### PR TITLE
Disable changelog fetching in pipeline lib

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkins-infra/helpdesk/issues/2
and
https://github.com/jenkinsci/docker-agent/issues/229

### CODE REVIEW

File: dist/profile/files/buildmaster/pipeline-configuration.groovy [hunk 1]
```diff
@@ -21,6 +21,8 @@ LibraryConfiguration lib = new LibraryConfiguration('pipeline-library',
 
 lib.implicit = true
 lib.defaultVersion = 'master'
+lib.includeInChangesets = false
+lib.cachingConfiguration = new LibraryCachingConfiguration(180, null)
```
Comment (timja) [inline dist/profile/files/buildmaster/pipeline-configuration.groovy:25]: relatively new feature thought we may as well try it out

**Review by lemeurherve — APPROVED** [review]

File: dist/profile/files/buildmaster/pipeline-configuration.groovy (hunk 1 above)
Comment (lemeurherve) [reply dist/profile/files/buildmaster/pipeline-configuration.groovy:25]: FTR, https://javadoc.jenkins.io/plugin/workflow-cps-global-lib/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration.html

**Comment by dduportal** [comment]
Merging to unblock tag re-push

File: dist/profile/files/buildmaster/pipeline-configuration.groovy (hunk 1 above)
Comment (dduportal) [reply dist/profile/files/buildmaster/pipeline-configuration.groovy:25]: For info: https://github.com/jenkins-infra/helpdesk/issues/2751

**Pull request merged**